scarlet2/plot.py

```diff
@@ -657,7 +656,6 @@ def sources(
         norm=None,
         channel_map=None,
         show_model=True,
-        show_hallucination=False,
```

Comment:
Will want to remove `show_hallucination` from the docstring as well.

Reply:
I thought I had done that...